config_ctrl.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference path="/usr/share/grafana/public/app/headers/common.d.ts" />
```

Comment:
This should definitely be a relative path. Is it really needed? Actually I made it work without imports and reference. Please remove those.